.github/workflows/publish.yml

```diff
@@ -15,8 +14,7 @@ jobs:
     steps:
       # Checks-out the repository under $GITHUB_WORKSPACE, so it's accessible to the job
       - uses: actions/checkout@v2
-
-      - name: Build and publish a collection
-        uses: artis3n/ansible_galaxy_collection@v2
+      - name: galaxy
+        uses: robertdebock/galaxy-action@1.2.0
```

Comment:
We should reference a specific SHA when using third-party actions.

Reply:
done

Reply:
by quick glance into our other collections  - for instance pulp one - it also uses tags instead of SHA - something that should be raised and planned - to search all our repos's GitHub actions that are using tags